### PR TITLE
Added Russian language

### PR DESCRIPTION
**Pull request by eksploit**

### CODE REVIEW

**Review by denysdovhan — CHANGES_REQUESTED** [review]
Please, read contribution guide: https://github.com/denysdovhan/purifier-card/blob/master/CONTRIBUTING.md#how-to-add-translation

You must include your translation in `src/localize.js`

**Comment by eksploit** [comment]
> 
> 
> Please, read contribution guide: https://github.com/denysdovhan/purifier-card/blob/master/CONTRIBUTING.md#how-to-add-translation
> 
> You must include your translation in `src/localize.js`

Sorry, did not notice, fixed

File: src/localize.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ import * as bg from './translations/bg.json';
 import * as zh from './translations/zh.json';
 import * as de from './translations/de.json';
 import * as ca from './translations/ca.json';
+import * as ca from './translations/ru.json';
```
Comment (denysdovhan) [inline src/localize.js]:
```suggestion
import * as ru from './translations/ru.json';
```

**Review by denysdovhan — CHANGES_REQUESTED** [review]

**Review by denysdovhan — APPROVED** [review]

**Comment by denysdovhan** [comment]
Thanks for contributing!

**Pull request merged**